51-android.rules

```diff
@@ -165,6 +165,7 @@ ATTR{idProduct}=="5201", ENV{adb_fast}="yes"
 ATTR{idProduct}=="5203", ENV{adb_adb}="yes"
 ATTR{idProduct}=="5208", ENV{adb_adb}="yes"
 
+ATTR{idProduct}=="2d00", ENV{adb_adb}="yes"
```

Comment:
I'm unsure of the differences between `adb_adb` and e.g. `adb_adbfast` and what fits best here. I've tried this one first and it works great